### ISSUE TITLE
Keylogger

### ISSUE THREAD

**Issue by Guest565-jpg** [ahtihska/SpotifyPremium#14]
Dont download this, I recently downloaded this shit and when I tried to log in my Steam account that had the same email got hacked and it spammed to all my friends some stupidass fake link, I have proofs also so go fuck yourself

**Comment by MorkinPorkin** [comment]
Yeah, I've also gotten this but thankfully my account is fully secured, I have 2FA and everything, I'm trying my best to remove everything sus on my pc. Especially with Task Manager, I fully recommend to not install this

**Comment by SmellsGoood** [comment]
Lol would have been better to read it here. Unfortunately too late. I'm resetting my whole computer and have already changed my password. There are really disgusting people on this earth who have nothing better to do than to cause harm
